python/ava-tests.py

```diff
@@ -24,7 +24,12 @@
 
 ENCODED_VIDEO_EXTENSIONS = (".mp4", ".mov", ".mkv", ".avi")
 # TODO: figure out how to handle this. On osx the /usr/bin/time is not the GNU version.
+
 GNU_TIME = True
+cmd = "time -v"
```

Comment:
This won't work in general. At least in linux, the time in-bash function has priority over the GNU time command. That's why you need to set the full path. 

If this doesn't work in Mac, we need a solution

Reply:
I have the same issue on my Fedora. It has the same time as my Mac.